src/main/java/Car.java

```diff
@@ -70,7 +70,7 @@ public void newLocation(double x, double y, double distanceIn) {
      */
     @Override
     public String toString() {
-        return "Car #:" + id + "\tTime traveled: " + time;
+        return (id + 1) + "\t\t" + String.format("%.1f", time);
```

Comment:
What does String.format... do? I see that id+1 makes it so that it's car 1 instead of car 0.